auth/oauth/types.go

```diff
@@ -149,6 +157,9 @@ type AuthorizationServerMetadata struct {
 	// If omitted, the default value is true. (hence pointer, or add custom unmarshalling)
 	PresentationDefinitionUriSupported *bool `json:"presentation_definition_uri_supported,omitempty"`
 
+	// RequireSignedRequestObject specifies if the authorization server requires the use of signed request objects.
+	RequireSignedRequestObject bool `json:"require_signed_request_object,omitempty"`
```

Comment:
I believe Sphereon does not use this (https://github.com/search?q=repo%3ASphereon-Opensource%2FSSI-SDK%20require_signed_request_object&type=code).

Are there instances in which we know for sure we have to use it?

Reply:
When using DIDs, it's required. (at least to get an acceptable level of security)

Reply:
Ah and if it's not specified in the metadata, it defaults to false, which is fine then